py/desispec/workflow/procfuncs.py

```diff
@@ -1158,7 +1164,7 @@ def submit_tilenight(ptable, prows, calibjobs, internal_id, queue, reservation,
 def science_joint_fit(ptable, sciences, internal_id, queue='realtime', reservation=None,
                       z_submit_types=None, dry_run=0, strictly_successful=False,
                       check_for_outputs=True, resubmit_partial_complete=True,
-                      system_name=None):
+                      system_name=None,use_gpuspecter=False):
```

Comment:
This doesn't seem to be passed on to the `joint_fit` function here (this is just a wrapper). You did add the hooks to `joint_fit`, so you just need to add the argument in the function call on line 1178.

Reply:
Thanks for catching this. As I understand it, jobs processing multiple exposures (other than tilenights) don't involve the extraction step. So does anything called by `checkfor_and_submit_joint_job` even need to know about what kind of specter to use?  If not, I'll just get rid of `use_gpuspecter` in the functions `checkfor_and_submit_joint_job`, `science_joint_fit`, and `joint_fit`.

Reply:
@akremin I went ahead and got rid of `use_specter` in the functions for "joint" fits and jobs, since they don't involve extractions. Please check this to make sure we are not missing it anywhere after this change.